recipe/build.sh

```diff
@@ -5,11 +5,16 @@ set -ex
 GOBIN=$(go env GOBIN)
 export GOBIN=$GOBIN
 
+go get -d github.com/google/go-licenses
+# Copy our patched vendored source before starting the build.
+cp -r \
+  licenseclassifier-${licenseclassifier_commit}/* \
+  "${GOPATH:-"$( go env GOPATH )"}"/pkg/mod/github.com/google/licenseclassifier@*-${licenseclassifier_commit}/
```

Comment:
This feels rather awkward... Well, I'm no Go dev and can't tell what other solution would look better :shrug:.

Reply:
I intend to upstream this and also the path for `go-licenses` -- but that involves signing a CLA for Google and as such can take a bit of time.
Let's go with the patches for now and clean up afterwards.